Dockerfile.local

```diff
@@ -0,0 +1,10 @@
+# Build the binary
+FROM golang:1.17
```

Comment:
This Dockerfile isn't building anything as it stands (i.e. doesn't need the go toolchain).

Reply:
Yeah agreed - if this is just a dev Dockerfile, we could also just use a scratch base image too. There might be some static vs. dynamically linked issues with this approach but it should produce a real small container image using that base image.

Reply:
Good call, probably a smaller container here makes more sense.